NEWS.md

```diff
@@ -9,7 +9,7 @@
 * Provides `instance-storage-bulk 1.0`
 * Provides `instance-date-types 1.0`
 * Provides `locations 3.1`
-* Provides `instance-storage 10.3`
```

Comment:
As we remove property - it should be 11.0. It's breaking change

Reply:
Also you should check where in other APIs instance schema is used and also bump its version